LoopKitUI/Views/Settings Editors/InsulinModelSelection.swift

```diff
@@ -46,17 +46,20 @@ public struct InsulinModelSelection: View, HorizontalSizeClassOverride {
     var glucoseUnit: HKUnit
     var supportedModelSettings: SupportedInsulinModelSettings
     let appName: String
+    let mode: PresentationMode
     
     public init(
         viewModel: InsulinModelSelectionViewModel,
         glucoseUnit: HKUnit,
         supportedModelSettings: SupportedInsulinModelSettings,
-        appName: String
+        appName: String,
```

Comment:
(I think `appName` should come from the environment; I'm going to be touching up our shared SwiftUI environment stuff after this PR so I'll see if I can do it)

Reply:
Thanks! I'll add a todo